docs/src/paradox/03-apis/api-v2/tei-xml.md

```diff
@@ -0,0 +1,433 @@
+                <xsl:value-of select="$authorFamilyNameText"/>, <xsl:value-of
+                    select="$authorGivenNameText"/>
+            </persName>
+
+            <xsl:variable name="dateValue" select="//beol:creationDate/@rdf:resource"/>
+
+            <xsl:variable name="dateObj"
+                          select="//knora-api:DateValue[@rdf:about=$dateValue]"/>
+
+            <xsl:copy-of select="knora-api:dateformat($dateObj)"/>
+
+        </correspAction>
+    </xsl:template>
+
+    <xsl:template match="beol:letter/beol:hasRecipientValue">
+        <xsl:variable name="recipientValue" select="@rdf:resource"/>
+
+        <xsl:variable name="recipientIAFValue"
+                      select="//knora-api:LinkValue[@rdf:about=$recipientValue]//beol:hasIAFIdentifier/@rdf:resource"/>
+        <xsl:variable name="recipientFamilyNameValue"
+                      select="//knora-api:LinkValue[@rdf:about=$recipientValue]//beol:hasFamilyName/@rdf:resource"/>
+        <xsl:variable name="recipientGivenNameValue"
+                      select="//knora-api:LinkValue[@rdf:about=$recipientValue]//beol:hasGivenName/@rdf:resource"/>
+
+        <correspAction type="received">
+
+            <xsl:variable name="recipientIAFText"
+                          select="//knora-api:TextValue[@rdf:about=$recipientIAFValue]/knora-api:valueAsString/text()"/>
+            <xsl:variable name="recipientFamilyNameText"
+                          select="//knora-api:TextValue[@rdf:about=$recipientFamilyNameValue]/knora-api:valueAsString/text()"/>
+            <xsl:variable name="recipientGivenNameText"
+                          select="//knora-api:TextValue[@rdf:about=$recipientGivenNameValue]/knora-api:valueAsString/text()"/>
+
+            <persName>
+                <xsl:attribute name="ref"><xsl:value-of select="knora-api:iaf($recipientIAFText)"
+                /></xsl:attribute>
+                <xsl:value-of select="$recipientFamilyNameText"/>, <xsl:value-of
+                    select="$recipientGivenNameText"/>
+            </persName>
+
+        </correspAction>
+    </xsl:template>
+
+    <!-- ignore text if there is no template for the element containing it -->
+    <xsl:template match="text()"> </xsl:template>
+
+
+</xsl:transform>
+```
+
+The complete request looks like this:
+
+```
+HTTP GET request to http://host/v2/tei/resourceIri&textProperty=textPropertyIri&mappingIri=mappingIri&gravsearchTemplateIri=gravsearchTemplateIri&teiHeaderXSLTIri=teiHeaderXSLTIri
+```
+
+See `webapi/src/it/scala/org/knora/webapi/e2e/v1/KnoraSipiIntegrationV1ITSpec.scala` for a complete test case involving the sample data ("create a mapping for standoff conversion to TEI referring to an XSLT and also create a Gravsearch template and an XSLT for transforming TEI header data").
+
+When you provide a custom conversion, it is up to you to ensure the validity of the TEI document. You can use this service to validate: <http://teibyexample.org/xquery/TBEvalidator.xq>.
```

Comment:
Please make this a Markdown link.

Reply:
done in https://github.com/dhlab-basel/Knora/pull/862/commits/670ab38f9277c2bc7874457c93c5e032357bb114